### DISCUSSION TITLE
How can I obtain the http.server.active_requests metric?

### DISCUSSION THREAD

**Discussion by zerocooldog** [open-telemetry/opentelemetry-java-instrumentation#12810] [Q&A] [answered]
I found that the HttpServerExperimentalMetrics class provides experimental support for the http.server.active_requests metric. However, I am unable to receive this metric on the collector. I'm testing with a Spring Boot application by injecting the Java agent.

Do I need to configure something in the config.properties file? Here are the settings I've configured:

```
otel.instrumentation.spring-webmvc.enabled=true
otel.instrumentation.spring-web.enabled=true

```

Or, does it require using Apache or Nginx instead of Tomcat to get this metric?

**Comment by laurit** [comment, marked as answer]
See https://opentelemetry.io/docs/zero-code/java/agent/instrumentation/http/#enabling-experimental-http-telemetry `otel.instrumentation.http.server.emit-experimental-telemetry` is the one you need.

**Reply by zerocooldog** [reply]
@laurit I always appreciate your answers.
